content/pt-br/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -0,0 +1,431 @@
+---
+title: kubeadm init
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+Este comando inicializa um nó da camada de gerenciamento do Kubernetes.
+
+<!-- body -->
+
+{{< include "generated/kubeadm_init.md" >}}
+
+### Fluxo do comando Init {#init-workflow}
+
+O comando `kubeadm init` inicializa um nó da camada de gerenciamento do Kubernetes
+através da execução dos passos abaixo:
+
+1. Roda uma série de verificações pré-execução para validar o estado do sistema
+   antes de efetuar mudanças. Algumas verificações emitem apenas avisos, outras
+   são consideradas erros e cancelam a execução do kubeadm até que o problema
+   seja corrigido ou que o usuário especifique a opção
+   `--ignore-preflight-errors=<lista-de-erros-a-ignorar>`.
+
+1. Gera uma autoridade de certificação (CA) auto-assinada para criar identidades
+   para cada um dos componentes do cluster. O usuário pode informar seu próprio
+   certificado CA e/ou chave ao instalar estes arquivos no diretório de
+   certificados configurado através da opção `--cert-dir` (por padrão, este
+   diretório é `/etc/kubernetes/pki`).
+   Os certificados do servidor da API terão entradas adicionais para nomes
+   alternativos (_subject alternative names_, ou SANs) especificados através da
+   opção `--apiserver-cert-extra-sans`. Estes argumentos serão modificados para
+   caixa baixa quando necessário.
```

Comment:
I understood, but "caracteres minúsculos" instead of "caixa baixa" becomes clearer to me. Think about that "Estes argumentos serão modificados para caracteres minúsculos quando necessário"

Reply:
Fixed!